privaterelay/templates/includes/banners.html

```diff
@@ -21,6 +21,20 @@
     </div>
   </div>
 {% else %}
+  <div class="c-data-notification-banner js-notification">
+    <div class="dismiss-wrapper">
+      <button class="data-notification-dismiss js-dismiss"><img src="{% static 'images/x-close.svg' %}" alt="Close"></button>
+    </div>
+    <div class="data-notification-banner-bg">
+      <div class="data-notification-banner-content">
+        <div class="data-notification-banner-copy">
+          <p class="data-notification-banner-header">{% ftlmsg 'banner-label-data-notification-header' %}</p>
+          <p class="data-notification-banner-sub">{% ftlmsg 'banner-label-data-notification-body' %}</p>
+          <a class="data-notification-banner-link" href="#" target="_blank" rel="noopener noreferrer">{% ftlmsg 'banner-label-data-notification-cta' %}</a>
```

Comment:
This will be the final link:

```suggestion
          <a class="data-notification-banner-link" href="/accounts/settings" target="_blank" rel="noopener noreferrer">{% ftlmsg 'banner-label-data-notification-cta' %}</a>
```

(Btw, does it need to open in a new tab?)